corfudb-tools/src/main/java/org/corfudb/browser/CorfuBrowser.java

```diff
@@ -0,0 +1,59 @@
+package org.corfudb.browser;
+
+import com.google.common.reflect.TypeToken;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.CorfuTable;
+import org.corfudb.runtime.collections.DynamicCorfuKey;
+import org.corfudb.runtime.collections.DynamicCorfuRecord;
+import org.corfudb.util.serializer.DynamicProtobufSerializer;
+import org.corfudb.util.serializer.ISerializer;
+import org.corfudb.util.serializer.Serializers;
+
+@Slf4j
+public class CorfuBrowser {
+    private CorfuRuntime runtime;
+    String singleNodeEndpoint;
+
+    public CorfuBrowser(String host, int port) {
+        singleNodeEndpoint = String.format("%s:%d", host, port);
+        runtime = createRuntime(singleNodeEndpoint);
+    }
+
+    private CorfuRuntime createRuntime(String endpoint) {
+        runtime = new CorfuRuntime(endpoint).setCacheDisabled(true).connect();
+        return runtime;
+    }
+
+    public CorfuTable getTable(String namespace, String tableName) {
+        final int code = 25;
```

Comment:
This number will get difficult to keep track of. We should keep this at some common location.

Reply:
Yes.  One way is to keep a configuration file with all constant parameters used by corfu.  Right now I have created a constant in this file itself.  I can work on this cleanup in parallel.